packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -557,10 +563,8 @@ export default class SchedulerTask {
 
                     const blocks = getChartThresholdAlertBlocks({
                         ...getBlocksArgs,
-                        footerMarkdown: `This is a ${thresholdFooter} sent by Lightdash. For security reasons, delivered files expire after ${
-                            this.s3Client.getExpirationWarning()?.days || 3
-                        } days`,
-                        imageUrl,
+                        footerMarkdown: `This is a ${thresholdFooter} sent by Lightdash.`,
+                        imageUrl: slackImageUrl,
```

Comment:
should we keep an expiration text if the image is hosted on s3?

this can be a return type for `tryUploadingImageToSlack`

`{ url: string, expiring: boolean }`

Reply:
done